zerver/tests/test_auth_backends.py

```diff
@@ -1393,7 +1393,7 @@ def social_auth_test(self, account_data_dict: Dict[str, str],
         if next:
             self.assertEqual(data['next'], next)
         if is_signup:
-            self.assertEqual(data['is_signup'], is_signup)
+            self.assertEqual(data['is_signup'], '1')
```

Comment:
Here. There was a reason for it to be '1' instead of is_signup?

Reply:
I think the code is correct.  The `is_signup` flag in the test suite is a boolean, but it's reflecting on a bit of data in the social auth state where `'1'` is the encoding of true.  

I think it'd probably be a good idea to convert that field to use a boolean as well, but all we changed in this PR is how we tell the test to do registration or login.

Reply:
Aah I see :)